### PR TITLE
Buttons removed in "Sent" tab

### LINKED ISSUE #184: no action takes place on tapping the buttons
#### Software and hardware versions 
Android vx.x.x, device used...

#### Problem description
When tapped on "Send" and "Select all", no action takes place.

#### Steps to reproduce the problem

sent

#### Expected behavior
Maybe some activity can be associated with the respective buttons **OR**
 it can be removed from there.
#### Other information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by iamvasuh**
Closes #184 

#### What has been done to verify that this works as intended?
I made the app run on a device and it's showing the expected result.
The two buttons "Send" and "Select All" are removed as there was no use of them.

#### Why is this the best possible solution? Were any other approaches considered?
This was the best possible solution for the result to be seen and redundancy to be eliminated

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
It improves the UI/UX.
sent1

#### Before submitting this PR, please make sure you have:
- [ ] run `./gradlew pmd checkstyle lint findbugs` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

**Comment by iamvasuh** [comment]
@shobhitagarwal1612  @lakshyagupta21 plz have a look.

File: skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ private void setupAdapter() {
 
     private void setEmptyViewVisibility(String text) {
         if (transferInstanceList.size() > 0) {
-            buttonLayout.setVisibility(View.VISIBLE);
+            buttonLayout.setVisibility(View.INVISIBLE);
```
Comment (shobhitagarwal1612) [inline skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java]: If button layout is not going to be visible in this fragment, then why not do it just once in onCreateView?

Comment (iamvasuh) [reply skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java]: there will be 2-3 lines of code to be removed if I do it from onCreate

Comment (shobhitagarwal1612) [reply skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java]: I agree that your choice of editing least number of lines was reasonable. But that creates a redundancy which must be resolved

Comment (iamvasuh) [reply skunkworks_crow/src/main/java/org/odk/share/fragments/SentInstancesFragment.java]: yes sure

**Comment by iamvasuh** [comment]
@shobhitagarwal1612 @lakshyagupta21  plz have a look on the update

**Pull request merged**